src/handlers/meta_transaction_handlers.ts

```diff
@@ -199,6 +203,11 @@ function parseRequestParams(req: express.Request): MetaTransactionQuoteRequestPa
         ]);
     }
 
+    const priceImpactProtectionPercentage =
+        req.query.priceImpactProtectionPercentage === undefined
+            ? DEFAULT_PRICE_IMPACT_PROTECTION_PERCENTAGE
+            : Number.parseFloat(req.query.priceImpactProtectionPercentage as string);
```

Comment:
With parseFloat here - Will the user need to supply ?p=0.315 or ?p=31.5 for setting prime impact protection to be higher than 31.5% ?

Reply:
`?p=0.315` would be 31.5%. I think this is consistent with our other "percentage" input params